_posts/2022-10-31-trino-summit-2022-teaser-3.md

```diff
@@ -0,0 +1,236 @@
+more to both virtual and in-person attendees! This will be done during our
+sponsored giveaway breaks between sessions where we challenge both online and 
+virtual attendees in a race against time to bag the swag! 
+
+### 3. Federate 'em all
+
+This year's summit will be a free event that federates both data and humans. The
+theme extends from a popular show that many of you know called Pokémon. To
+understand the connection here, let's break down what we mean by federate 'em
+all. In the same way Pokémon protagonist, Ash Ketchum, catches and trains
+heterogeneous creatures called Pokémon, Trino queries and filters heterogeneous
+data sets from various data sources. 
+
+If you're not familiar with Pokémon, a losing strategy is to train just one or
+two Pokémon as different types of Pokémon are better suited to different tasks.
+In the same way, centralizing all of your data to a single data warehouse or
+data lake doesn't make sense either. There will be different use cases and 
+different needs across the company. Rather than spending your time building
+brittle one-size-fits-all architectures, Trino enables you to connect to
+[multiple data sources]({{site.url}}/docs/current/connector.html) using ANSI SQL.
+
+For in-person attendees, another fun swag prize is that we will be throwing out
+poke-stress balls and Pokémon trading card booster packs for audience
+participation! Check out the teaser for this year's Trino Summit and get ready
+to ***Federate 'em all***!
+
+<iframe src="https://www.youtube.com/embed/o2MJvRKG14M" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allowfullscreen="">
+</iframe>
+
+### 4. Experience beautiful San Francisco
+
+For those attending in-person, you will get to enjoy the beautiful San Francisco
+area. The venue we booked (back in 2020) is the
+[Commonwealth club](https://www.starburst.io/info/trinosummit/#location), right
+on the San Francisco Bay. The building is beautiful with a large auditorium for
+the main event, and plenty of floors and rooms for socializing.
+
+<p align="center">
+<img align="center" width="50%" src="/assets/blog/trino-summit-2022/commonwealth-club.jpeg"/><br/>
+</p>
+
+At the end of the summit, we will have a happy hour on the scenic roof-deck that
+gazes over the San Francisco bay at the iconic Oakland Bay Bridge.
+
+<p align="center">
+<img align="center" width="75%" src="/assets/blog/trino-summit-2022/san-francisco.png"/><br/>
+</p>
+
+We know this only applies to our in-person attendees, but remember if you join
+us virtually, there are still plenty of resources to network and interact 
+throughout the conference. We will be taking questions from our virtual audience
+and there will also be a chat forum to discuss with attendees from across the
+globe. Plus, unlike those of us attending in-person, no travel required and
+pants are optional during the event!
```

Comment:
Not sure if this is okay ... maybe drop to be on the safe side

Reply:
I'll change it to pjs are optional...